examples/localai-webui/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.6'
+
+services:
+  api:
+    image: quay.io/go-skynet/local-ai:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8080:8080
+    env_file:
+      - .env
+    volumes:
+      - ./models:/models:cached
+    command: ["/usr/bin/local-ai"]
+
+  frontend:
+    image: dhruvgera/localai-frontend:latest
```

Comment:
this is looking good, but didn't tried it out yet: can we have the image hosted internally or make the compose file build the image otherwise?

Reply:
Let me set up the image pushing in the frontend repository

Reply:
images created at : `quay.io/go-skynet/localai-frontend:master`!

Reply:
Comitted! Please merge is no further conflicts or changes are required